report/data_sources/hubspot/client.py

```diff
@@ -38,6 +42,27 @@ def chunk(items: Iterable, chunk_size: int):
     return iter(lambda: tuple(islice(iter_items, chunk_size)), ())
 
 
+def chunk_with_max_len(items: Iterable[str], chunk_size: int, max_chars: int):
+    """Return items in chunks with a maximum size and char length.
+
+    :param items: Items to break into chunks
+    :param max_chars: Maximum total char length of a single chunk
+    :param chunk_size: The maximum size of a chunk
+    """
+    batch, batch_chars = [], 0
+
+    for item in items:
+        batch_chars += len(item)
```

Comment:
Interesting, there's no overhead for separating the values or other parts of the query?